packages/common/src/api/library.ts

```diff
@@ -3,7 +3,7 @@ import type { full } from '@audius/sdk'
 import { AudiusQueryContextType, createApi } from '~/audius-query'
 import { UserCollectionMetadata } from '~/models/Collection'
 import { Kind } from '~/models/Kind'
-import { makeActivity } from '~/services/audius-api-client/ResponseAdapter'
+import { makeActivity } from '~/services/audius-api-client'
```

Comment:
this was split out bc it pulled in libs somehow, but may not be needed anymore bc i think it was bc of dayjs, which is also in web now